colors/_functions.scss

```diff
@@ -1,7 +1,7 @@
 @function getColor($args...) {
-	@return oColorsGetPaletteColor($args...);
+	@return oColorsByName($args...);
 }
 
 @function getColorFor($args...) {
-	@return oColorsGetColorFor($args...);
+	@return oColorsByUsecase($args...);
```

Comment:
Good opportunity to get rid of `getColorFor` and `getColor`? 
`oColorsByUsecase` is different to `oColorsFor`, so there will be some 
migration steps to document here.

Reply:
Let's do it, rip the plaster off!